concent_api/core/models.py

```diff
@@ -20,7 +20,7 @@ class StoredMessage(Model):
     type        = PositiveSmallIntegerField()
     timestamp   = DateTimeField()
     data        = BinaryField()
-    task_id     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, blank = False)
+    task_id     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, null = True, blank = True)
```

Comment:
Is there a reason to change to `blank = True` ?

Reply:
We were getting:
```
File "/home/jakub/Documents/work/projects/concent/concent_api/core/views.py", line 1111, in store_message_and_message_status
    golem_message.full_clean()
  File "/home/jakub/.virtualenvs/consent/lib/python3.5/site-packages/django/db/models/base.py", line 1250, in full_clean
    raise ValidationError(errors)
django.core.exceptions.ValidationError: {'task_id': ['This field cannot be blank.']}
```
because of `golem_message.full_clean()`

Reply:
What is value set on `task_id` which cause this error ? Is it None or '' ?

Reply:
We checked both :/